source/frontend/src/components/maps/leaflet/Map.test.tsx

```diff
@@ -157,7 +156,7 @@ xdescribe('MapProperties View', () => {
     );
     mockGetParcel = jest.fn(async () => ({} as IProperty));
 
-    (useApi as unknown as jest.Mock<Partial<typeof useApi>>).mockReturnValue({
+    (useApiGeocoder as unknown as jest.Mock<Partial<typeof useApiGeocoder>>).mockReturnValue({
```

Comment:
this seems a bit odd since loadProperties and getProperty are not something that was returned from useApi in the past. Perhaps this isn't being used?

Reply:
Yes looks like, all the test cases in this file are also skipped.